variables.tf

```diff
@@ -581,3 +581,9 @@ variable "create_static_kubeconfig" {
   type        = bool
   default     = true
 }
+
+variable "cluster_node_pool_mode" {
+  description = "Flag for predefined cluster node configurations - Values : default, minimal"
+  type        = string
+  default     = "default"
```

Comment:
So to create 'minimal' you would do: default = "minimal"? Seems like default is always a default and you need a var that says 'deployment type' - which defaults to default and can be set to 'minimum'?